src/styles/_variables.scss

```diff
@@ -8,8 +10,8 @@ $extra-large-desktop-breakpoint: 2560px;
 /*                          Colours                         */
 
 //colors
+$button-text-color: #f3f2e8;
 $text-color: #000000;
-$background-color: #1f1b1b;
 
```

Comment:
let's add the 

`$background-color: #1f1b1b;`

line back in, as it is being used in a couple of places

Reply:
added